packages/wcg-crypto/index.d.ts

```diff
@@ -0,0 +1,13 @@
+export class WCGCrypto implements Crypto {
```

Comment:
Think I'd add the triple-slash directive in every file that makes use of WinterCG globals